Inoffizielles Skript.lyx

```diff
@@ -3047,15 +3047,15 @@ thlabel{euler}
 
 \end_inset
 
- In einem zusammenhängenden, planaren Graph 
+ In einem zusammenhängendem, planarem, einfachem, nichtleerem Graphen 
```

Comment:
„nicht leer“ und |V| > 0 sind (vor allem in Verbindung mit „zusammenhängend“ synonym